### DISCUSSION TITLE
CHARTS AND GRAPHS LIKE SHOPIFY

### DISCUSSION THREAD

**Discussion by vitalijalbu** [medusajs/medusa#5780] [Feature Requests]
WOULD BE GREAT HAVING CHARTS AND GRAPHS LIKE SHOPIFY

![83d50640-62c6-4a4a-bf29-60e3e82b8db0](https://github.com/medusajs/medusa/assets/40239902/8f41c876-0979-47ae-956c-81faaedfce87)

**Comment by luis-cruzt** [comment]
Hey!

Take a look to this:
https://github.com/medusajs/medusa/discussions/5955

**Comment by radoslaw-sz** [comment]
hi @vitalijalbu - you may want to check: https://github.com/RSC-Labs/medusa-store-analytics
